<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WateringINPImport</class>
 <widget class="QDialog" name="WateringINPImport">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watering INP Import</string>
  </property>
  <widget class="QLabel" name="selectFileLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>16</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Select File:</string>
   </property>
  </widget>
  <widget class="QgsFileWidget" name="newINPDirectory" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectCrsLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>66</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Source CRS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectCRSButton">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>90</y>
     <width>23</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="selectedCRSLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="convertINPFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>